doc/nitrocli.1

```diff
@@ -292,13 +330,27 @@ for the \fBuser\fR type.
 .TP
 .B NITROCLI_PASSWORD
 A password used by commands that require one (e.g., \fBhidden open\fR).
+.P
+Additionally, these environment variables can be used to overwrite the
+configuration (see the Config file section):
+.TP
+.B NITROCLI_MODEL
+Restrict connections to the given device model (string, default none, see
+\fB\-\-model\fR).
 .TP
 .B NITROCLI_NO_CACHE
-If this variable is present in the environment, do not cache any inquired
-secrets using \fBgpg\-agent\fR(1) but ask for them each time they are needed.
-Note that this variable does not cause any cached secrets to be cleared. If a
-secret is already in the cache it will be ignored, but left otherwise untouched.
-Use the \fBpin clear\fR command to clear secrets from the cache.
+If set to true, do not cache any inquired secrets (boolean, default:
```

Comment:
Have you checked whether `true` is really the only option? Would `1` cause us to error out?

Can/should we keep the verbiage about not clearing existing cached secrets? That is, why did you remove so much?

Reply:
Yes, 1 causes an error: `provided string was not true or false while parsing value '1' provided by NO_CACHE`.

As I introduced a `--no-cache` option, I moved the detailed explanation to the documentation for that option.  The documentation for the environment variable refers to the option for more information.